src/pesum.c

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: GPLv2
+/*
+ * pesum.c - pesum command line tool
+ * Copyright Peter Jones <pjones@redhat.com>
+ */
+
+#include "fix_coverity.h"
+
+#include <err.h>
+#include <popt.h>
+
+#include <nss.h>
+#include <prerror.h>
+
+#include "pesign.h"
+#include "pesign_standalone.h"
+
+static struct {
+	int flag;
+	const char *name;
+} flag_names[] = {
+	{DAEMONIZE, "daemonize"},
```

Comment:
flag_names and print_flag_name() don't seem to be used for anything.

Reply:
This is used in common code, but has to be defined in the front end (for fairly dumb reasons in this case.) That's something that could stand to be cleaned up at some point.